### DISCUSSION TITLE
about completion type

### DISCUSSION THREAD

**Discussion by iangithub** [microsoft/semantic-kernel#2756] [Q&A]
Hi ;

Is SK  support chatcompletion and can use the system and user roles in chatcompletion ?
In the prompt template, only the completion type of dialogue can be used .

**Comment by evchaki** [comment]
@iangithub you can use system and user roles now - https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/KernelSyntaxExamples/Example63_ChatCompletionPrompts.cs
